packages/perseus/src/util/interactive.ts

```diff
@@ -196,14 +196,11 @@ _.extend(GraphUtils.Graphie.prototype, {
      * Get mouse coordinates in pixels
      */
     getMousePx: function (event: Readonly<{pageX?: number; pageY?: number}>) {
-        const graphie = this;
+        // @ts-expect-error - TS2532 - Object is possibly 'undefined'.
```

Comment:
Which object is possible undefined, `el`? Can we add a check to narrow to a defined type?